src/eureka/lib/gaussian_min.py

```diff
@@ -13,16 +13,16 @@ def evalgauss(params, x, y, x0, y0):
     Parameters
     ----------
     params : list
-        List components : amplitude, x_stddev, y_stddev. 
-        Acts as the inital guess from meta (params_guess).
+             List components : amplitude, x_stddev, y_stddev. 
```

Comment:
Similar to what I said about centerdriver.py, all these description lines are also over-indented now